### DISCUSSION TITLE
Percy is not rendering custom web components

### DISCUSSION THREAD

**Discussion by Th3S4mur41** [percy/percy-storybook#540] [Support] [answered]
We're using storybook to document custom Web Components developed with [Lit](https://lit.dev/).
While storybook is rendering those components just fine, the snapshots in percy are just blank pages.

Tried both to `percy storybook:start` and `percy storybook ./dist` without success.

Any help would be appreciated

**Comment by Robdel12** [comment, marked as answer]
Hey @Th3S4mur41! You'll want to enable JavaScript: https://docs.percy.io/docs/storybook#unexpected-diffs

Percy can't serialize the shadow DOM due to a lack of web APIs to support it: https://github.com/percy/cli/issues/280#issuecomment-824226395 thankfully this is the storybook SDK so enabling JavaScript will work just fine.
